src/test/fuzz/coinscache_sim.cpp

```diff
@@ -151,10 +151,6 @@ class CoinsViewBottom final : public CCoinsView
     {
         auto it = m_data.find(outpoint);
         if (it == m_data.end()) {
-            if ((outpoint.n % 5) == 3) {
```

Comment:
In commit 6243ceb65b83ea8dc3a689fdcc868a3a94b86e80 ` refactor: Remove unrealistic simulation state `

The comment above should be removed in this commit as well, since it is no longer accurate.

Reply:
Done